test/module.js

```diff
@@ -561,8 +565,6 @@ const cases = {
   'grid-flow-row-dense': { 'grid-auto-flow': 'row dense' },
   'grid-flow-col-dense': { 'grid-auto-flow': 'column dense' },
 
-  'max-w-screen-sm': { 'max-width': theme.screens['sm'] },
-
```

Comment:
Moved further up in the file, just a house-keeping thing.

Reply:
Awesome, thank you 🙇